admin/howto/support.md

```diff
@@ -1,52 +1,31 @@
 # Get support
 
-Hub engineers can provide support and debugging for issues that are related to the 2i2c Hub infrastructure. There are a few ways to get in touch listed below (in decreasing order of preference!)
+For most issues, the Community Champion of a hub should be able to solve the problem on their own.
+Community Champions have **administrator rights** on a JupyterHub, and this comes with a number of tools and privileges that allow them to debug and assist others.
 
-(support:issues)=
-## Via GitHub Issues
-
-We handle most support requests via [GitHub issues in the `pilot/` repository](https://github.com/2i2c-org/pilot/issues).
-This ensures that most support (and the resulting fixes) are done in a public place.
-
-To help you open the right issue, here are a few buttons you can click to quickly open a support request in the `pilot/` repository:
-
-````{panels}
-:container: full-width
-:column: col-4
-:card: border-1 text-center
-Request an improvement ✨
-^^^
-```{link-button} https://github.com/2i2c-org/pilot/issues/new?labels=enhancement&template=tech-request.md
-:text: Update/add packages, infrastructure, etc.
-:classes: stretched-link
-
----
-Report an issue 🐛
-^^^
-```{link-button} https://github.com/2i2c-org/pilot/issues/new?labels=bug&template=tech-support.md
-:text: Crashing sessions, broken webpages, etc.
-:classes: stretched-link
----
-Ask a question ❓
-^^^
-```{link-button} https://github.com/2i2c-org/pilot/issues/new?labels=question&template=questions.md
-:classes: stretched-link
-:text: Generic questions about the hub.
-```
-````
+If a problem persists after attempting to solve it, or if the hub requires a change that the Community Champion cannot implement themselves, they should reach out to the 2i2c support team.
+Hub engineers can provide support and debugging for issues that are related to the 2i2c Hub infrastructure.
+There are a few ways to get in touch listed below (in decreasing order of preference!)
 
-:::{note}
-If you have a support question that shouldn't be in public, please [send a support email](support:email).
-:::
+(support:issues)=
+## File a support ticket
 
-(support:slack)=
-## Connect on Slack
+We handle most support requests via [GitHub issues in the `support/` repository](https://github.com/2i2c-org/support/issues).
+This is a **private repository** that is accessible to Community Champions and 2i2c Engineers.
 
-You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the pilot repository](https://github.com/2i2c-org/pilot/issues/new) and we're happy to give you access.
+[**Click here to file a support ticket**](https://github.com/2i2c-org/support/issues/new) or go to [support.2i2c.org](http://support.2i2c.org).
 
+:::{note}
```

Comment:
You are saying the same again a little bit below... it feels a little bit repetitive.